.github/ISSUE_TEMPLATE/1_Bug_report.md

```diff
@@ -11,12 +11,15 @@ labels: Bug
 <!-- A clear and concise description of the problem. -->
 
 **How to reproduce**
-<!-- Code and/or config needed to reproduce the problem. If it's a complex bug,
-     create a "bug reproducer" as explained in:
+<!--  ⚠️  This is the most important part of the report ⚠️
+     Without a way to easily reproduce your issue, there is little chance we will be able to help you and work on a fix.
+     Please, take the time to show us some code and/or config that is needed for others to reproduce the problem.
+     If it's a complex bug, create a "bug reproducer" as explained in:
```

Comment:
Maybe we should suggest to create a bug reproducer by default? I often see people posting code fragments which would be enough to reproduce something, but still requires someone to try it out to copy and paste these snippets into an application. That's something that takes time and we could make it more clear IMO that making it as easy as possible to reproduce an issue increases the likelihood for someone to look into it.

Though I have no idea how to phrase that in a short way.